src/utils/proxify.ts

```diff
@@ -1,4 +1,8 @@
-const proxyUrl = 'https://proxy.dial.to';
+let proxyUrl = 'https://proxy.dial.to';
+
+export function setProxyUrl(url: string) {
+  proxyUrl = url;
```

Comment:
Hey! Since it's now possible to override proxyUrl, could you please add checks in `proxify` or in `setProxyUrl` function to check if `proxyUrl` is a valid url and not empty. Something like this would be ideal:

```ts
const DEFAULT_PROXY_URL = 'https://proxy.dial.to';
let proxyUrl = DEFAULT_PROXY_URL;

export function setProxyUrl(url: string) {
  if (!url || !isURL(url)) {
    console.warn('Invalid proxy url, resetting'); // for devs
    proxyUrl = DEFAULT_PROXY_URL;
    return;
  }

  proxyUrl = url;
}
```
or it can be a ternary, your call, just needs some checks.